src/containers/audio-rewards-page/components/modals/ModalDrawer.tsx

```diff
@@ -53,8 +53,15 @@ const ModalDrawer = (props: ModalDrawerProps) => {
       dismissOnClickOutside={props.dismissOnClickOutside}
       title={props.title}
       contentHorizontalPadding={props.contentHorizontalPadding}
-      titleClassName={gradientTitle ? styles.modalTitle : undefined}
-      headerContainerClassName={gradientTitle ? styles.modalHeader : undefined}
+      titleClassName={cn(props.titleClassName ? props.titleClassName : '', {
```

Comment:
no need to change, but I usually do `className={cn({ [props.titleClassName!]: !!props.titleClassName })}` in case you were wondering